src/walk/index.js

```diff
@@ -162,13 +162,38 @@ function ignore(_node, _st, _c) {}
 
 export const base = {}
 
-base.Program = base.BlockStatement = (node, st, c) => {
-  for (let stmt of node.body)
-    c(stmt, st, "Statement")
+const moduleDeclarationTypes = [
+  "ImportDeclaration",
+  "ExportNamedDeclaration",
+  "ExportDefaultDeclaration",
+  "ExportAllDeclaration"
+]
+base.Program = (node, st, c) => {
+  for (let child of node.body)
+    if (moduleDeclarationTypes.includes(child.type)) {
```

Comment:
We're targeting ES3 runtimes, to `includes` (and even `Array.indexOf`) shouldn't be used.